bin/webextension-toolbox

```diff
@@ -61,11 +70,15 @@ function logCompileOutput (error, stats) {
     version: false,
     hash: false
   }))
+
+  if (options.verbose) {
+    console.error('\nCompilation finished\n')
+  }
 }
 
 function logBanner () {
   const banner = `
-           ╔════════╗                     
```

Comment:
My IDE is configured to automatically remove trailing whitespace. I can revert this if needed.

Reply:
For me, it worked fine, but if it causes an issue, I will add it back. :slightly_smiling_face: